include/config.hpp

```diff
@@ -48,6 +48,12 @@
  */
 #define BUZZER_PIN 27
 
+/**
+ * Maximum throttle percentage for arming the motors
+ * The throttle stick must be less than this percentage to arm the motors
+ */
+#define MAX_ARMING_THROTTLE_PERCENTAGE 0.05
```

Comment:
Should this be `MIN_ARMING_THROTTLE_PERCENTAGE`? (min instead of max)

Reply:
Think max is right because it will still arm if percentage throttle is less than 5%